.travis.yml

```diff
@@ -56,7 +56,7 @@ script:
 - export TAICHI_REPO_DIR=$TRAVIS_BUILD_DIR
 - export PYTHONPATH=$TAICHI_REPO_DIR/python
 - export PATH=$TAICHI_REPO_DIR/bin:$PATH
-- ti test -v && cd python && $PYTHON build.py try_upload
+- ti test -t 2 -v && cd python && $PYTHON build.py try_upload
```

Comment:
Maybe we should let multiprocessing.cpu_count to auto-detect how much cores CI have? Or you may want to set `threads = 2` for `cpucount == 1`?

Reply:
`multiprocessing.cpu_count` sometimes crashes on the buildbots, unfortunately.

> Or you may want to set `threads = 2` for `cpucount == 1`?

You mean hyperthreading?

Reply:
Sorry to hear that, may due to VMs, can keep this for now.
Yes, hyperthreading cpus want this to ultilize their resources.

Reply:
Sounds good.